deployability/modules/workflow_engine/examples/agent/vagrant/test-agent-basic-info-vagrant.yaml

```diff
@@ -0,0 +1,203 @@
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: create
+          - provider: "{macos-infra-provider}"
+          - size: small
+          - composite-name: "{agent}"
+          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
+          - track-output: "{working-dir}/agent-{agent}/track.yaml"
+          - label-termination-date: "1d"
+          - label-team: "qa"
+          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
+    on-error: "abort-all"
+    foreach:
+      - variable: macos-agent-os
+        as: agent
+    cleanup:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: delete
+          - track-output: "{working-dir}/agent-{agent}/track.yaml"
+
+  # Unique manager provision task
+  - task: "provision-manager-{manager-os}"
+    description: "Provision the manager."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/provision/main.py
+          - inventory: "{working-dir}/manager-{manager-os}/inventory.yaml"
+          - install:
+            - component: wazuh-manager
+              type: assistant
+              version: 4.7.4
+              live: True
+    depends-on:
+      - "allocate-manager-{manager-os}"
+    on-error: "abort-all"
+
+  # Generic agent test task
+  - task: "run-agent-{agent}-tests"
+    description: "Run tests install for the agent {agent}."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/testing/main.py
+          - targets:
+            - wazuh-1: "{working-dir}/manager-{manager-os}/inventory.yaml"
+            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
+          - tests: "install,registration,basic_info"
+          - component: "agent"
+          - wazuh-version: "4.7.4"
+          - wazuh-revision: "40717"
+          - live: "True"
+    foreach:
+      - variable: agent-os
+        as: agent
+    depends-on:
+      - "allocate-agent-{agent}"
+      - "provision-manager-{manager-os}"
+
+  # Generic macOS agent test task
+  - task: "run-agent-{agent}-tests"
+    description: "Run tests install for the agent {agent}."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/testing/main.py
+          - targets:
+            - wazuh-1: "{working-dir}/manager-{manager-os}/inventory.yaml"
+            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
+          - tests: "install,registration,basic_info,connection,restart,stop,uninstall"
+          - component: "agent"
+          - wazuh-version: "4.7.4"
+          - wazuh-revision: "40717"
+          - live: "True"
+    foreach:
+      - variable: macos-agent-os
+        as: agent
+    depends-on:
+      - "allocate-macos-agent-{agent}"
+      - "provision-manager-{manager-os}"
```

Comment:
Add line break

Reply:
Done in [9e922d9](https://github.com/wazuh/wazuh-qa/pull/5407/commits/9e922d98bfb106b811f91a29940f703366345787)